SPECS/python-setuptools/python-setuptools.spec

```diff
@@ -19,14 +19,15 @@ Source0:        https://pypi.org/packages/source/s/setuptools/setuptools-%{versi
 %package -n python3-setuptools
 Summary:        %{summary}
 
-Requires:       python3-devel
+Requires:       python3
```

Comment:
Move this Requires below after BuildRequires entries.

Reply:
will change